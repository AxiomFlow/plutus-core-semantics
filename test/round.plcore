(module Foo (imported) (exported () ()) (local () ())
  (define f1 (builtin round (2.7)))
  (define f2 (builtin round (2.3)))
  )
