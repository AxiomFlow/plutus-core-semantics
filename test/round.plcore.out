<T> <k> . </k> <lenv> .Map </lenv> <genv> ( Foo . f1 ) |-> 3 ( Foo . f2 ) |-> 2 ( Foo . f3 ) |-> 0 </genv> <mod> Foo </mod> <sym> .Map </sym> </T>
